api/apis/job_handler.go

```diff
@@ -0,0 +1,66 @@
+package apis
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"code.cloudfoundry.org/cf-k8s-controllers/api/presenter"
+
+	"github.com/go-logr/logr"
+	"github.com/gorilla/mux"
+)
+
+const (
+	JobGetEndpoint  = "/v3/jobs/{guid}"
+	syncSpacePrefix = "sync-space.apply_manifest-"
```

Comment:
I know we chatted about this a bit on Slack, but wanted to put some context here since I know this looks weird:

`space.apply_manifest` is the operation that these jobs get assigned by Cloud Controller:
https://github.com/cloudfoundry/cloud_controller_ng/blob/4a23a419862ef9b7272f411a4435e33fc7305f30/app/jobs/space_apply_manifest_action_job.rb#L56

I proposed appending `sync-` to it because this is a synchronous manifest application and used hyphens because those align with the general guid structure.

Open to other suggestions or thoughts on what a good sentinel value here could be as well.

Reply:
With that context, I don't think there's much we can do. We could use an underscore, but the guid would still have hyphens.

Reply:
Looks like we would always end up with a mixture of hypens, underscores and dashes. I'll leave the implementation as is.